microprofile/server/src/main/java/io/helidon/microprofile/server/Server.java

```diff
@@ -526,6 +374,32 @@ public Builder addApplication(Application application) {
             return this;
         }
 
+        /**
+         * If any application or resource class is added through this builder, applications discovered by CDI are ignored.
+         * You can change this behavior by setting the retain discovered applications to {@code true}.
```

Comment:
I don't know what this sentence means.

Reply:
If you use `Server.Builder` and explicitly add a JAX-RS application or JAX-RS resource class to it, then classes discovered via CDI mechanisms will be ignored (e.g. all JAX-RS applications with bean defining annotations and all JAX-RS resources in bean archives).
If you set the flag to `true`, we will retain both the "discovered" and "explicitly configured" applications and resource classes.

Reply:
Updated the comment in javadoc.

Reply:
The part I was confused about was: `by setting the retain discovered applications to true`.  Is this a configuration setting?  Is it something else?